model/src/rollover.rs

```diff
@@ -20,6 +20,15 @@ pub enum Version {
     V2,
 }
 
+impl std::fmt::Display for Version {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
```

Comment:
Nit: In other parts of the codebase we do `use std::fmt;` and refer to these symbols as `fmt::Display` to make things a bit shorter.

Reply:
I had it first but then I didn't want to add the extra import for this 🙃